easybuild/easyblocks/i/impi.py

```diff
@@ -141,6 +143,22 @@ def post_install_step(self):
             for script in [os.path.join(script_path, 'mpivars.sh') for script_path in script_paths]:
                 apply_regex_substitutions(os.path.join(self.installdir, script), regex_subs)
 
+        # Recompile libfabric if necessary
+        if impiver >= LooseVersion('2019.0.0') and self.cfg['rebuild_libfabric']:
+            if self.cfg['ofi_internal']:
+                os.chdir(os.path.join(self.installdir, 'libfabric'))
```

Comment:
please use `change_dir` from `easybuild.tools.filetools` (which spits out a proper error message in case the `chdir` fails)